src/accelerate/components/List.tsx

```diff
@@ -39,6 +40,9 @@ export default function List( props: Props ) {
 
 	const maxViews = posts.reduce( ( carry, post ) => Math.max( post.views, carry ), 0 );
 
+	const [currentPage, setCurrentPage] = useState(1);
```

Comment:
Spacing
```suggestion
	const [ currentPage, setCurrentPage ] = useState( 1 );
```

Reply:
Oversight from my side, when I was implementing the new pagination I kept the old pagination in place, removed and resolved now.